docs/source/release.myst

```diff
@@ -1,35 +1,8 @@
-# Mitiq Git flow
-
-The basic development workflow for Mitiq is done in units of milestones.
-These are tracked in the GitHub milestone feature and all issues that
-are planned to be addressed in the current milestone should be tagged
-with the proper milestone.
-
-All releases for Mitiq are tagged on the `master` branch with tags for
-the version number of the release. Development work is done on separate
-branches and forks that get merged into `master` when they are ready to
-be included in the next release.
-
-The main steps of our git flow are as follows:
--   Feature work and bug fixes are done on branches (external
-    contributors should fork and then work on branches on their fork)
--   Once work is ready for review and inclusion in a release, make a PR
-    from the branch/fork to master on the Mitiq repo. Squash merges are
-    preferred.
--   PRs are then reviewed by the team and the community and then merged
-    into master as appropriate. This means that this feature/fix will be
-    included in the next release.
--   When it is time to make a release, a PR is made from the master
-    branch to the release branch and final automatic testing and manual
-    review is done to make sure it is good to be released.
--   Once the code is ready to be released, the PR from Master to release
-    is approved and a tag is created on release for the appropriate
-    semantic version number.
-
 # Releasing a new version of Mitiq
 
 ```{note}
 These instructions are for Mitiq maintainers.
+Releasing a new version of Mitiq is typically performed by the milestone manager whose responsibilities are detailed [here](https://mitiq.readthedocs.io/en/stable/milestone_managing.html).
```

Comment:
```suggestion
Releasing a new version of Mitiq is typically performed by the milestone manager whose responsibilities are detailed {ref}`here <source/milestone_managing>` 
```

Reply:
I tried this and couldn't get it to work. I think I probably needed to add an anchor for it to link to? Weird that you can't link to a page with this syntax. Anyway, while playing around, I found that you _can_ use the typical local file reference within myst!

Reply:
Oh I actually completely missed your first comment! I see how to do it now. I think the local file reference is a little cleaner since it doesn't require you to link to a header (what if the header changes), and doesn't require adding an anchor.

Still, good to know! I don't think I knew about the `ref` syntax.

Reply:
Also since we set `myst_heading_anchors = 3` in [docs/source/conf.py](https://github.com/unitaryfund/mitiq/blob/56a77209474b2d3c258c7ce300cd36453646f77e/docs/source/conf.py#L115), all the anchors are automatically generated ([documentation](https://myst-parser.readthedocs.io/en/latest/syntax/optional.html#syntax-header-anchors)), so you can use the typical markdown link syntax, just FYI!